TicTacXOs/src/styles/components.css

```diff
@@ -1,3 +1,9 @@
+#logo {
```

Comment:
this was for a pic of the logo i used in the static website, has to be changed bc now it is text not a photo
font info is on figma

Reply:
also you should add a font categoru